packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -698,6 +708,10 @@ export interface SdkClientAccessor<TServiceOperation extends SdkServiceOperation
   response: SdkClientType<TServiceOperation>;
 }
 
+export interface SdkClientInitializationMethod extends SdkMethodBase {
+  kind: "clientinitialization";
+}
```

Comment:
Another method for Java emitter to filter out.

Reply:
it is not in the `methods` list. you could use as you will.